### PR TITLE
build: prepare the build configuration for alchemist-web-renderer

### PR DESCRIPTION
**Pull request by AngeloFilaseta**

### CODE REVIEW

File: alchemist-web-renderer/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+
+group = it.unibo.alchemist
+artifactId = alchemist-web-renderer
+projectLongName = Alchemist Web Renderer
+projectDescription = Alchemist module that starts a web server and allows to visualize an environment on a web browser.
+
+kotlin.code.style=official
+kotlin.mpp.stability.nowarn=true
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=2g -Xmx2g
```
Comment (DanySK) [inline alchemist-web-renderer/gradle.properties:17]: Do you really need a 2g metaspace?

File: settings.gradle.kts [hunk 2]
```diff
@@ -35,7 +35,8 @@ include(
     "alchemist-test",
     "alchemist-ui-tooling",
     "alchemist-swingui",
-    "alchemist-fxui"
+    "alchemist-fxui",
+    "alchemist-web-renderer"
```
Comment (DanySK) [inline settings.gradle.kts:39]:
```suggestion
    "alchemist-web-renderer",
```

Trailing comma

**Review by DanySK — CHANGES_REQUESTED** [review]
one question and a minor edit. Great job, keep it up.

File: alchemist-web-renderer/gradle.properties (hunk 1 above)
Comment (AngeloFilaseta) [reply alchemist-web-renderer/gradle.properties:17]: I thought the configuration was present in a suggested fullstack-jvm-js multiplatform project I found in the documentation but I was wrong. I probably used this when i tried to make some jvm to js conversion in the first phase of the project and forgot it there. I removed it as the project perfectly works without it.

**Pull request merged**